rosplan_planning_system/CMakeLists.txt

```diff
@@ -148,3 +149,28 @@ install(DIRECTORY include/rosplan_action_interface
 install(DIRECTORY common/bin
         DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/common/bin/
         FILE_PERMISSIONS OWNER_EXECUTE)
+
+
+#############
+##  Tests  ##
+#############
+
+
+if (CATKIN_ENABLE_TESTING)
+    find_package(rostest REQUIRED)
+    find_package(roslib REQUIRED)
+    # Planner Interface
+    add_rostest_gtest(PlannerInterfaceTests test/launch/planner_interface_tests.test test/src/PlannerInterfaceTests.cpp)
+    target_link_libraries(PlannerInterfaceTests ${catkin_LIBRARIES})
```

Comment:
in general if you notice you are linking to catkin_LIBRARIES and not to rostest_LIBRARIES so the instruction find_package(rostest REQUIRED) , is nice because it ensures user has this package but then later on is not used. It is cleaner if you add rostest as a catkin required pkg, under : ```(catkin REQUIRED COMPONENTS``` , which is good because actually rostest is a ros pkg (you can roscd into it)